crypto/evp_extra/p_ed25519.c

```diff
@@ -104,4 +104,6 @@ const EVP_PKEY_METHOD ed25519_pkey_meth = {
     NULL /* derive */,
     NULL /* paramgen */,
     NULL /* ctrl */,
+    NULL /* encapuslate */,
```

Comment:
There are a few more places where we define `EVP_PKEY_METHOD`s, where we want the explicit `NULL` e.g. `/crypto/fipsmodule/evp/p_{ec,rsa}.c`